### ISSUE TITLE
Rbind subdomain request with redirecting steps to github pages?

### ISSUE THREAD

**Issue by saadjavaid** [rbind/support#969]
## Netlify website address

None yet, I want my saad.rbind.io subdomain to redirect to saadjavaid.github.io pages. Would it be possible please?

## Preferred rbind.io subdomain

saad.rbind.io

### Agreement

- [x] By submitting this request, I promise I will at least write one blog post or create one web page on my website after I get the rbind.io subdomain.

**Comment by nanxstats** [comment]
Yes, for GitHub Pages, please put a `CNAME` file in your repo to make it work, for example: https://github.com/rbind/support/issues/139.

@saadjavaid Do you prefer us to proceed with adding the subdomain on our side?
